server/strategy.go

```diff
@@ -903,18 +910,9 @@ func rebootOps(c *cke.Cluster, constraints *cke.Constraints, rebootArgs DecideOp
 			ops = append(ops, op.RebootDrainStartOp(nf.HealthyAPIServer(), rebootArgs.NewlyDrained, &c.Reboot))
 		}
 	}
-	if len(rebootArgs.DrainCompleted) > 0 {
-		phaseReboot = true
-		ops = append(ops, op.RebootRebootOp(nf.HealthyAPIServer(), rebootArgs.DrainCompleted, &c.Reboot))
-	}
 	if len(rebootArgs.DrainTimedout) > 0 {
-		phaseReboot = true
 		ops = append(ops, op.RebootDrainTimeoutOp(rebootArgs.DrainTimedout))
 	}
-	if len(rebootArgs.RebootDequeued) > 0 {
-		phaseReboot = true
-		ops = append(ops, op.RebootDequeueOp(rebootArgs.RebootDequeued))
-	}
 	if len(ops) > 0 {
 		phaseReboot = true
 	}
```

Comment:
Thank you for fixing `phaseReboot`!

I checked this variable carefully, and I was convinced that this function need not return `phaseReboot`.
This function should only determine `ops` from the given information. 

The caller(`DecideOps()`) should determine the phase. And which can be determined only by the returned `ops`.

Could you change the return value of this function  and change the caller as following?
It's okay to change it in another PR.

https://github.com/cybozu-go/cke/blob/v1.28.0/server/strategy.go#L106-L113
```
	// 11. Reboot nodes if reboot request has been arrived to the reboot queue, and the number of unreachable nodes is less than a threshold.
	if ops := rebootOps(c, constraints, rebootArgs, nf); len(ops) > 0 {
		if !nf.EtcdIsGood() {
			log.Warn("cannot reboot nodes because etcd cluster is not responding and in-sync", nil)
			return nil, cke.PhaseRebootNodes
		}
		return ops, cke.PhaseRebootNodes
	}
```